includes/discovery/libvirt-vminfo.inc.php

```diff
@@ -9,18 +9,23 @@
 
     $ssh_ok = 0;
 
+    $userHostname = $device['hostname'];
+    if (!is_null($config['libvirt_username'])) {
```

Comment:
$config['libvirt_username'] is predefined in the defaults with a NULL-value. I personally do not like empty() because is also returns true on zero (which might be the value you want to use)